src/common/Network/PacketDef/Zone/ServerZoneDef.h

```diff
@@ -283,6 +283,105 @@ namespace Sapphire::Network::Packets::Server
     char unk2[0x4]; // This has probs something to do with the support desk (inquiry id?)
   };
 
+  struct FFFXIVIpcMarketBoardItemListingCount : FFXIVIpcBasePacket< MarketBoardItemListingCount >
+  {
+    uint32_t itemCatalogId;
+    uint32_t unknown1; // does some shit if nonzero
+    uint16_t requestId;
+    uint16_t quantity; // high/low u8s read separately?
+    uint32_t unknown3;
+  };
+
+  struct FFXIVIpcMarketBoardItemListing : FFXIVIpcBasePacket< MarketBoardItemListing >
+  {
+    struct ItemListing // 152 bytes each
+    {
+      uint64_t listingId;
+      uint64_t retainerId;
+      uint64_t retainerOwnerId;
+      uint64_t artisanId;
```

Comment:
while correct, the actual name for this is `contentId`

Reply:
I wonder if SE uses the same ids for retainers too, can probably leave this as is actually

Reply:
Ids of `retainerId` is exactly the same as the one in [NetworkRetainerSummary](https://github.com/menphina/Thaliak.Network/blob/master/Thaliak.Network/Messages/NetworkRetainerSummary.cs) when the item is listed by the retainer.

Reply:
I don't have much study on the intrinsic connection of these ids or how SE stores these data. So `contentId` should be a better name.

Reply:
makes sense - SE uses u64s to identify players globally (as in, cross datacenter) and it makes sense that retainers would be using the same IDs as well